src/views/DashboardView.vue

```diff
@@ -71,22 +70,134 @@
 import { mapState } from 'pinia'
 import { useUserStore } from '../stores/user'
 import NeedSignIn from '../components/shared/NeedSignIn.vue'
+import TheWorkout from '../components/shared/TheWorkout.vue'
 import TheExercise from '../components/shared/TheExercise.vue'
+import TheButton from '../components/shared/TheButton.vue'
+import TheToast from '../components/shared/TheToast.vue'
+import { supabase } from '../supabase'
 
 export default {
   name: 'DashboardView',
   components: {
     NeedSignIn,
-    TheExercise
+    TheWorkout,
+    TheExercise,
+    TheButton,
+    TheToast
+  },
+  data() {
+    return {
+      userName: '',
+      userID: '',
+      userWorkouts: [],
```

Comment:
Are there other workouts beside user Workouts?

Reply:
Yeah, there are also `allWorkouts` because the thing is I couldn't assign workouts to a particular user, so - since I needed this assignment only in Dashboard - I decided to first assign the user ID to each created workout and then to fetch all workouts (`allWorkouts`) and displaying only those, whose user ID matches the ID of the actual user ID (`userWorkouts`).

So, basically, `userWorkouts` are those that were created by this particular user, whereas `allWorkouts` are all workouts ever created by all users.